selftest/tcpconnect/tcpserver

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+nc -F -l -p 12345 > /dev/null
```

Comment:
Same with `-F`. Looks like there's an openbsd, gnu, and 'classic' netcats. A licensing issue.

Reply:
Same here.